tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -157,27 +158,29 @@ pub fn command(s: &str) -> Result<Command, cli::Error> {
                     .with(arguments())
                     .map(|args| {
                         // strip quotes from file paths.
-                        let mut sargs = Vec::with_capacity(args.len());
+                        // not sure how to map this and still throw the error so doing it the old fashioned way
+                        let mut files = Vec::with_capacity(args.len());
                         for arg in args.iter() {
```

Comment:
This is usually written `for arg in &args`, since `(&args).into_iter()` is equivalent to `args.iter()`.